BlazorAccessibilityTests.Tests/AriaAccordionTests.razor

```diff
@@ -1,335 +1,197 @@
+                     <span class="Accordion-icon"></span>
+                   </span>
+                 </button>
+               </h3>
+               <div id:regex="[\d\w-]+" role="region" aria-labelledby:regex="[\d\w-]+" class="Accordion-panel">
+                 <div>
+                   <p>P1 Content</p>
+                 </div>
+               </div>
+               <h3>
+                 <button aria-expanded="false" class="Accordion-trigger" aria-controls:regex="[\d\w-]+" id:regex="[\d\w-]+">
+                   <span class="Accordion-title">P2
+                     <span class="Accordion-icon"></span>
+                   </span>
+                 </button>
+               </h3>
+               <div id:regex="[\d\w-]+" aria-labelledby:regex="[\d\w-]+" class="Accordion-panel" hidden="">
+                 <div>
+                   <p>P2 Content</p>
+                 </div>
+               </div>
+             </div>);
     }
 
-// Space or Enter
-// When focus is on the accordion header of a collapsed section, expands the section.
-// Note: The Space or Enter buttons trigger an "onclick" event 
-
-// aria-expanded="true"	button	
-// Set to true when the Accordion panel is expanded, otherwise set to false.
-
+    // Space or Enter
+    // When focus is on the accordion header of a collapsed section, expands the section.
+    // Note: The Space or Enter buttons trigger an "onclick" event 
+    // aria-expanded="true"	button	
+    // Set to true when the Accordion panel is expanded, otherwise set to false.
     [Fact(DisplayName = "Set to true when the Accordion panel is expanded, otherwise set to false.")]
     public void AriaExpanedAttribute()
     {
-        var cut = Render(@<AriaAccordion>
-                                <AccordionPanel Title="P1">
-                                    <p>P1 Content</p>
-                                </AccordionPanel>
-                                <AccordionPanel Title="P2">
-                                    <p>P2 Content</p>
-                                </AccordionPanel>
-    </AriaAccordion>
-    );
-
-        var panels = cut.FindComponents<AccordionPanel>();
-        var button1 = panels[0].Find("button");
-        var button2 = panels[1].Find("button");
-
-
-        button1.GetAttribute("aria-expanded").Should().Be("true");
-        button2.GetAttribute("aria-expanded").Should().Be("false");
-        panels[0].Instance.IsExpanded.Should().BeTrue();
-        panels[1].Instance.IsExpanded.Should().BeFalse();
+        var cut = RenderAccordionWithTwoPanels();
+        var button1 = cut.Find("h3:nth-of-type(1) > button");
+        var button2 = cut.Find("h3:nth-of-type(2) > button");
 
         button2.Click();
 
         button1.GetAttribute("aria-expanded").Should().Be("false");
         button2.GetAttribute("aria-expanded").Should().Be("true");
-        panels[0].Instance.IsExpanded.Should().BeFalse();
-        panels[1].Instance.IsExpanded.Should().BeTrue();
```

Comment:
I would consider `IsExpanded` internal state of the component, so I suggest not coupling tests to it.